src/components/App.js

```diff
@@ -2,14 +2,16 @@ import React, { Component } from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
 
 import Login from './../containers/login/LoginPage.jsx';
+import Header from './../containers/Header/header.jsx';
 
 class App extends Component {
   render() {
     return (
       <Router>
         <div>
-          <Route exact path="/" component={Login} />
-          <Route path="/login" component={Login} />
+    
+          <Route exact path="/" component={Header} />
```

Comment:
You don't need to change this route to header. Since according to the design, the header is part of some other components, just submit the components without displaying as routes.